metrics-monitoring/README.adoc

```diff
@@ -63,44 +63,48 @@ As a developer or administrator, you can view metrics in {product} to visualize
 
-* `kafka_controller_kafkacontroller_offline_partitions_count`: Number of partitions without an active leader and that are therefore not writable or readable
+* `kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum`: Number of client connection creations per second for the cluster. Kafka clients use persistent connections to interact with brokers in the cluster. A constant high number of connection creations might indicate a client issue. The maximum connection creation rate is 100 per second.
 
-* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Incoming byte rate for the instance in the last five minutes
+* `kafka_topic:kafka_topic_partitions:count`: Number of topics in the cluster. This does not include internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of topics is 1000.
 
-* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Outgoing byte rate for the instance in the last five minutes
+* `kafka_topic:kafka_topic_partitions:sum`: Number of partitions across all topics in the cluster. This does not include internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000.
+--
 
-* `kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum`: Number of client connections to the instance
+Broker metrics::
++
+--
+* `kafka_broker_quota_softlimitbytes`: Maximum amount of storage, in bytes, for this broker before producers are throttled. When this limit is reached, the broker starts throttling producers to prevent them from sending additional data. The maximum storage in the broker is 1000 GiB.
 
-* `kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum`: Number of attempted client connections per second for the instance
+* `kafka_broker_quota_totalstorageusedbytes`: Amount of storage, in bytes, that is currently used by partitions in the broker. The storage usage depends on the number and retention configurations of the partitions. This metric must stay below the `kafka_broker_quota_softlimitbytes` metric setting.
 
-* `kubelet_volume_stats_available_bytes`: Available disk space in the instance
+* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions in the cluster. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`. This count includes partitions from internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000.
 
-* `kubelet_volume_stats_used_bytes`: Used disk space in the instance
+* `kafka_controller_kafkacontroller_offline_partitions_count`: Number of partitions in the cluster that are currently offline. Offline partitions cannot be used by clients for producing or consuming data. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`.
+
```

Comment:
Agreed that the following two metrics are basically repeats of other metrics. Not sure what we should do here in docs if they're that way in the display.

Reply:
Yes they are basically repeats. They are computed from different components though so if you plot them they are close but not exactly the same. Maybe we can use the same descriptions?

Reply:
Hm, interesting. Well I'm not sure what to do here because which one do we tell users to use for which? If I point out for clarity that they are essentially the same, then users will want to know when to use which or if it genuinely doesn't make a difference. Do you know if it's true that users can use either metric as equally valid for the same purpose? If so, I suppose I could just mention that in the description. That way at least they're not wondering if it's their imagination.

Reply:
Hm, okay. So should I just say "This metric is similar to the X metric" ?